tftpsync/susemanager-tftpsync-recv/configure-tftpsync.sh

```diff
@@ -113,7 +117,7 @@ default_or_input "SUSE Manager Proxy IP Address" SUMA_PROXY_IP $(host $SUMA_PROX
 
```

Comment:
I didn't notice before because it was hidden by github, but I think the same should be done for proxy:

```bash
default_or_input "SUSE Manager Proxy IP Address" SUMA_PROXY_IP $(host $SUMA_PROXY_FQDN | awk '{ print $NF }' || echo "")
```